configure.ac

```
@@ -1212,7 +1222,18 @@ AS_IF([! $arch64],
 
 # Support for C11 atomic types
 
-OCAML_CC_SUPPORTS_ATOMIC([$cclibs])
+AS_CASE([$ocaml_cc_vendor],
+  [msvc-*],
+    [OCAML_CC_SUPPORTS_ATOMIC
+     AS_IF([! $cc_supports_atomic],
```

A thought: we could keep the line above (well, the `OCAML_CC_SUPPORS_ATOMIC([], [$cclibs])` version from below), which is the line that ultimately we expect to work on all platforms. Then:
```suggestion
AS_CASE([$cc_supports_atomic,$ocaml_cc_vendor],
  [false,msvc-*],
    [AS_IF([! $cc_supports_atomic],
```
(plus bracket adjustments)